src/pages/MainPage/index.jsx

```
@@ -31,7 +31,7 @@ export default function MainPage() {
           <Navbar />
           <Window isOpenWindow={isOpenWindow} setIsOpenWindow={setIsOpenWindow}>
             <Routes>
-              <Route path="/" element={<MapPage />} />
+              <Route path="/" element={<></>} />
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)